Spanner/tests/System/BackupTest.php

```diff
@@ -117,13 +121,17 @@ public function testCreateBackup()
     {
         $expireTime = new \DateTime('+7 hours');
         $versionTime = new \DateTime('-5 seconds');
+        $encryptionConfig = [
+            "encryptionType" => CreateBackupEncryptionConfig\EncryptionType::GOOGLE_DEFAULT_ENCRYPTION,
+        ];
 
         $backup = self::$instance->backup(self::$backupId1);
         $db1 = self::getDatabaseInstance(self::$dbName1);
 
         self::$createTime1 = gmdate('"Y-m-d\TH:i:s\Z"');
         $op = $backup->create(self::$dbName1, $expireTime, [
             "versionTime" => $versionTime,
+            "encryptionConfig" => $encryptionConfig,
```

Comment:
Just the smallest of nits, would we be able to use single quotes?

Reply:
Done. Been writing too much Python :sweat_smile: